src/current/_includes/v25.2/prod-deployment/insecure-test-load-balancing.md

```diff
@@ -12,12 +12,11 @@ For comprehensive guidance on benchmarking CockroachDB with TPC-C, see [Performa
 
     This should be a machine that is not running a CockroachDB node.
 
-1. Download the [CockroachDB archive](https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.linux-amd64.tgz) for Linux, and extract the binary:
+1. Download [CockroachDB for Linux]({% link {{ page.version.version }}/install-cockroachdb-linux.md %}), extract the binary, and copy it into the `PATH`:
```

Comment:
ah sorry i missed a couple of spots. @taroface and @jhlodin pls let me know what you think of the approach shown in the other files of linking everything to 'Install CockroachDB on Linux' and if you are on board I'll go through and update all of these usages as well